### PR TITLE
Opsgenie add config doc

### PR DESCRIPTION
**Pull request by BjMrq**

### CODE REVIEW

**Review by mok0230 — APPROVED** [review]

File: docs/modules/ROOT/pages/sentinel.adoc [hunk 1]
```diff
@@ -576,11 +576,39 @@ To avoid overwhelming the receiving webhook with many concurrent requests under
...
+
+* *Visible To* Teams and users that the alert will become visible to without sending any notification.type field is mandatory for each item, where possible values are team and user. In addition to the type field, either id or name should be given for teams and either id or username should be given for users. Please note: that alert will be visible to the teams that are specified within responders field by default, so there is no need to re-specify them within visibleTo field. You can refer below for example values (50 teams or users in total)
+
+* *Alias* Client-defined identifier of the alert, that is also the key element of https://support.atlassian.com/opsgenie/docs/what-is-alert-de-duplication/[Alert De-Duplication] (512 characters)
+
+* *Priority* Priority level of the alert. Possible values are P1, P2, P3, P4 and P5. Default value is P3
+
+* *Entity* Entity field of the alert that is generally used to specify which domain alert is related to (512 characters)
+
+* *Actions*	Custom actions that will be available for the alert (10 x 50 characters)
```
Comment (shahnami) [inline docs/modules/ROOT/pages/sentinel.adoc:597]: There's an whitespace here

**Review by shahnami — APPROVED** [review]
LGTM

**Review by zeljkoX — APPROVED** [review]

**Comment by zeljkoX** [comment]
@BjMrq should we merge this PR as it is already in prod?

**Comment by BjMrq** [comment]
> @BjMrq should we merge this PR as it is already in prod?

Thanks for the reminder!

**Pull request merged**